examples/vite-avatar-stack/index.html

```diff
@@ -1,19 +1,19 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap"
-      rel="stylesheet"
-    />
-    <title>Ably Realtime Examples - Avatar stack</title>
-  </head>
-  <body>
-    <div id="root"></div>
-    <script type="module" src="src/index.tsx"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap" rel="stylesheet" />
+  <title>Ably Realtime Examples - Avatar stack</title>
+</head>
+
+<body>
+  <div id="root"></div>
+  <script type="module" src="src/index.tsx"></script>
+</body>
+
+</html>
```

Comment:
hey @snikidev if you run `yarn format` from the root, it will revert back to the previous indentation - I'd suggest just making the favicon.ico change, this formatting change is incompatible with the current prettier settings